src/utilities/common.ts

```diff
@@ -1,7 +1,7 @@
 import * as constants from 'utilities/constants';
 import BigNumber from 'bignumber.js';
 import { getVaiTokenAddress } from './addressHelpers';
-import ethers from 'ethers';
+import * as ethers from 'ethers';
```

Comment:
import {ethers} from 'ethers';

Reply:
it was already fixed earlier

Reply:
These are equivalent but I am happy to use a named import